example/bloc_library/lib/main.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The Flutter Architecture Sample Authors. All rights reserved.
+// Use of this source code is governed by the MIT license that can be found
+// in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:bloc/bloc.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:todos_repository_flutter/todos_repository_flutter.dart';
+import 'package:flutter_architecture_samples/flutter_architecture_samples.dart';
+import 'package:bloc_library/localization.dart';
+import 'package:bloc_library/blocs/blocs.dart';
+import 'package:bloc_library/models/models.dart';
+import 'package:bloc_library/screens/screens.dart';
+
+void main() {
+  BlocSupervisor().delegate = SimpleBlocDelegate();
```

Comment:
Wat dis do? (Connected to other comment)

Reply:
It allows developers to create their own delegate which exposes things like global error handling as well as global transition handling. I’ll definitely update the docs to include more info 💯